afl-plot

```diff
@@ -196,14 +196,15 @@ exit 1
 fi
 
 mkdir -p "$outputdir/tmp"
+mkfifo "$outputdir/tmp/win_ids"
```

Comment:
can you please add a `rm -f "$outputdir/tmp/win_ids"` prior?
plus an `|| exit 1` on the mkfifo command?